net/BUILD.gn

```diff
@@ -3347,6 +3347,10 @@ test("net_unittests") {
 
       # Re-enable once we're on c++20
       "base/backoff_entry_serializer_unittest.cc",
+
+      # Fails on modular builds.
+      "base/proxy_string_util_unittest.cc",
```

Comment:
Do you want to just filter these tests in the test filters instead of removing them from the build?